test/test.transporters.ts

```diff
@@ -98,4 +98,42 @@ describe('Transporters', () => {
           done();
         });
   });
+
+  it('should use the http proxy if one is configured', async () => {
```

Comment:
nit: put these two tests in a describe section. Use something like the following to ensure you're not leaving side effects between/after tests:

```ts
let savedEnv;
beforeEach(() => {
  savedEnv = process.env;
  process.env = {};
});

afterEach(() => { process.env = savedEnv; });
```

Reply:
fixed!